benchmarking/benchmark.py

```diff
@@ -54,49 +54,101 @@ def get_times_skfp(X: np.ndarray, transformer_cls: type, **kwargs) -> np.ndarray
     return np.array(result).reshape((len(N_CORES), N_SPLITS))
 
 
-def save_results(
+LEGAL_PLOT_TYPES = ["time", "speedup", "fprints_per_second"]
+
+
+def save_plots(
     n_molecules: int,
     times: np.ndarray,
     title: str = "",
```

Comment:
I wouldn't make a default value here. This is pretty essential information, after all